app/src/main/java/org/connectbot/TerminalView.java

```diff
@@ -250,55 +237,18 @@ public boolean onSingleTapConfirmed(MotionEvent e) {
 		});
 	}
 
-	@TargetApi(11)
-	private void closeSelectionActionMode() {
-		if (selectionActionMode != null) {
-			selectionActionMode.finish();
-			selectionActionMode = null;
-		}
-	}
-
 	public void copyCurrentSelectionToClipboard() {
-		ClipboardManager clipboard =
-				(ClipboardManager) TerminalView.this.context.getSystemService(Context.CLIPBOARD_SERVICE);
-		if (currentSelection.length() != 0) {
-			clipboard.setText(currentSelection);
-		}
-
-		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.HONEYCOMB) {
-			closeSelectionActionMode();
-		}
-	}
-
-	@Override
-	protected void onSelectionChanged(int selStart, int selEnd) {
-		if (selStart <= selEnd) {
-			currentSelection = getText().toString().substring(selStart, selEnd);
-		}
-		super.onSelectionChanged(selStart, selEnd);
+		terminalTextViewOverlay.copyCurrentSelectionToClipboard();
```

Comment:
Added a null check. And yes it's still used when the user presses `Ctrl`+`Shift`+`C`